android/app/src/main/java/org/fossasia/openevent/fragments/FeedFragment.java

```diff
@@ -54,15 +54,16 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         View view = super.onCreateView(inflater, container, savedInstanceState);
 
         feedItems = new ArrayList<>();
+        feedFragmentViewModel = ViewModelProviders.of(this).get(FeedFragmentViewModel.class);
         RecyclerView.LayoutManager mLayoutManager = new LinearLayoutManager(getContext());
         feedRecyclerView.setLayoutManager(mLayoutManager);
-        feedAdapter = new FeedAdapter(getContext(), (FeedAdapter.OpenCommentsDialogListener)getActivity(), feedItems);
-        feedAdapter.setOnImageZoomListener((OnImageZoomListener)getActivity());
+        feedAdapter = new FeedAdapter(getContext(), (FeedAdapter.OpenCommentsDialogListener) getActivity(), feedItems);
+        feedAdapter.setOnImageZoomListener((OnImageZoomListener) getActivity());
```

Comment:
Don't cast. Either require the listener during construction or via setter, don't cast